charts/kubewarden/1.3.5/Chart.yaml-e

```diff
@@ -0,0 +1,16 @@
+annotations:
```

Comment:
What is this file???

Reply:
great question! It isn’t present in Jordon’s latest release 🤔  https://github.com/rancher/ui-plugin-charts/pull/39/files gonna clean it, it’s very odd… I am following https://confluence.suse.com/display/CU/Releasing+Official+Extensions that Jordon created… We have to manually sync the files since the `main`  branch is protected